quickannotator/api/v1/utils/shared_crud.py

```diff
@@ -0,0 +1,72 @@
+from datetime import datetime
+from sqlalchemy.orm import aliased, sessionmaker, Session, Query, DeclarativeBase
+import quickannotator.db as qadb
+from quickannotator.db import build_annotation_table_name, create_dynamic_model
+import shapely
+import json
+from sqlalchemy import func, text
+
+
+# TODO: Remove session from params
+def get_tile(session: Session, annotation_class_id: int, image_id: int, tile_id: int) -> qadb.Tile:
```

Comment:
inline with previous comments, its probably session, image id, class id, tileid in order of rate of change